### ISSUE TITLE
Add weather extension in calendar hovers pop up, like did in fluent theme

### ISSUE THREAD

**Issue by hackingguy** [vinceliuice/WhiteSur-gtk-theme#526]
Hi please add a weather extension in the calendar hovers popup, it will make it much more consistent like you did it in fluent theme!

![Screenshot from 2022-01-05 15-24-12](https://user-images.githubusercontent.com/28388650/148197836-ff5e8c73-8347-4ad1-b92c-19895185681b.png)

**Comment by kernelb00t** [comment]
I don't think this has the place here. This theme tries to be close to Apple's HIG and macOS design. If this is a really useful feature wanted by users, time can be invested in this. Anyway, feel free to put time in this yourself and try to implement it in the theme!
